src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,169 @@
+// React Imports
+import React from 'react';
+import { Link } from 'react-router-dom';
+// Solid Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import AppBar from '@mui/material/AppBar';
+import Badge from '@mui/material/Badge';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+// Custom Component Imports
+import NavMenu from './NavMenu';
+
+/**
+ * NavBar Component - Component that generates NavBar section for PASS
+ *
+ * @memberof GlobalComponents
+ * @name NavBar
+ */
+
+const NavBar = () => {
+  const { session } = useSession();
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [anchorElNotifications, setAnchorElNotifications] = React.useState(null);
+  const [openMenu, setOpenMenu] = React.useState(false);
+
+  const menuId = 'primary-search-account-menu';
+
+  const handleOpenMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+    setOpenMenu(true);
+  };
+
+  const handleNotificationsMenu = (event) => {
+    setAnchorElNotifications(event.currentTarget);
+  };
+
+  const handleNotificationsClose = () => {
+    setAnchorElNotifications(null);
+  };
+
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static" color="primary">
+        <Toolbar>
+          {session.info.isLoggedIn ? (
+            <>
+              <Typography variant="h4" noWrap component="div" mr="10px">
+                PASS
+              </Typography>
+              <>
+                <Link to="/PASS/home/">
+                  <Button
+                    variant="contained"
+                    aria-label="outlined primary button group"
+                    color="tertiary"
+                  >
+                    Home
+                  </Button>
+                </Link>
+                <Link to="/PASS/forms/">
+                  <Button
+                    variant="contained"
+                    aria-label="outlined primary button group"
+                    color="tertiary"
+                  >
+                    Forms
+                  </Button>
+                </Link>
+                <Box sx={{ flexGrow: 1 }} />
+                <IconButton
+                  size="large"
+                  aria-label="show 17 new notifications"
+                  color="inherit"
+                  onClick={handleNotificationsMenu}
+                >
+                  <Badge badgeContent={17} color="error">
+                    <NotificationsIcon />
+                  </Badge>
+                </IconButton>
+                <Menu
```

Comment:
Compared to NavMenu which have the Menu below the icon, the Notification Menu covers the Bell icon.

![Screen Shot 2023-05-09 at 14 40 31](https://github.com/codeforpdx/PASS/assets/14917816/b93ef951-8e1b-4a6e-9c13-8acf42a43101)
![Screen Shot 2023-05-09 at 14 35 18](https://github.com/codeforpdx/PASS/assets/14917816/05dc1d19-62e0-47cb-893a-1140013a0c37)

Make it below their respective icon and make sure they're both leveled at the same distance below the icon (I think you can use NavMenu as a guide for positioning that).

Reply:
Also, while it looks alright for demonstrative purposes, I think we can turn off the badgeContent for now. We don't exactly have any notifications implemented currently, so this will prevent some confusion.